README.md

```diff
@@ -308,147 +310,128 @@ Ukázka zaslané události:
 {
     "ProtocolVersion": 1,
     "DeviceId": "abc123",
-    "MessageType": "magnetic-detection-simple",
+    "DeviceType": "magnetic-detection-simple",
     "EventId": "c4056fc4-d433-4d2c-bb7f-23a691fd3dac",
     "EventTime": "2021-05-03T14:25:31.8437511Z",
     "EventType": "alive"
 }
 ```
 
 ## EventType transport
-Nastává při přechodu senzoru do transportního režimu, ke kterému dojde po vložení nové baterie nebo pomocí příkazu zaslaného prostřednictvím downlink API.
 
-Čidlo v transportním režimu má velmi nízkou spotřebu a neposílá žádné zprávy. Pro probuzení čidla z transportního režimu je třeba stisknout RESET tlačítko umístěné na plošném spoji čidla.
+Nastává při přechodu zařízení do transportního režimu, ke kterému dojde po vložení nové baterie nebo pomocí příkazu
+zaslaného prostřednictvím downlink API.
+
+Čidlo v transportním režimu má velmi nízkou spotřebu a neposílá žádné zprávy. Pro probuzení čidla z transportního režimu
+je třeba stisknout RESET tlačítko umístěné na plošném spoji čidla.
 
 Ukázka zaslané události:
 ```yaml
 {
     "ProtocolVersion": 1,
     "DeviceId": "abc123",
-    "MessageType": "magnetic-detection-simple",
+    "DeviceType": "magnetic-detection-simple",
     "EventId": "c4056fc4-d433-4d2c-bb7f-23a691fd3dac",
     "EventTime": "2021-05-03T14:25:31.8437511Z",
     "EventType": "transport"
 }
 ```
 
-## EventType downlink-acknowledge
-Informuje o doručení downlinkového commandu na zařízení.
+[//]: # (## EventType downlink-acknowledge)
 
-Ukázka zaslané události:
-```yaml
-{
-    "ProtocolVersion": 1,
-    "DeviceId": "abc123",
-    "MessageType": "magnetic-detection-simple",
-    "EventId": "c4056fc4-d433-4d2c-bb7f-23a691fd3dac",
-    "EventTime": "2021-05-03T14:25:31.8437511Z",
-    "EventType": "downlink-acknowledge"
-}
-```
+[//]: # (Informuje o doručení downlinkového commandu na zařízení.)
 
-## EventType tamper-open
-Nastává při rozepnutí tamperu - otevření krytu senzoru.
+[//]: # ()
 
-Ukázka zaslané události:
-```yaml
-{
-    "ProtocolVersion": 1,
-    "DeviceId": "abc123",
-    "MessageType": "magnetic-detection-simple",
-    "EventId": "c4056fc4-d433-4d2c-bb7f-23a691fd3dac",
-    "EventTime": "2021-05-03T14:25:31.8437511Z",
-    "EventType": "tamper-open"
-}
-```
+[//]: # (Ukázka zaslané události:)
+
+[//]: # (```yaml)
+
+[//]: # ({)
+
+[//]: # (    "ProtocolVersion": 1,)
+
+[//]: # (    "DeviceId": "abc123",)
+
+[//]: # (    "DeviceType": "magnetic-detection-simple",)
+
+[//]: # (    "EventId": "c4056fc4-d433-4d2c-bb7f-23a691fd3dac",)
+
+[//]: # (    "EventTime": "2021-05-03T14:25:31.8437511Z",)
+
+[//]: # (    "EventType": "downlink-acknowledge")
+
+[//]: # (})
+
+[//]: # (```)
```

Comment:
Nerozhodilo se tady nějak divně formátování?

Reply:
ne, ja jse to zakomentoval protoze v aktualni verzi nenabizime downlink.